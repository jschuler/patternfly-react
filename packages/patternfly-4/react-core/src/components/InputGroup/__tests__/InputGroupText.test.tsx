// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { InputGroupText } from '../InputGroupText';

describe('InputGroupText test', () => {
  it('InputGroupText should match snapshot', () => {
    const view = shallow(
      <InputGroupText
        className={"''"}
				children={<div>ReactNode</div>}
				component={<>ReactNode</>}
      />);
    expect(view).toMatchSnapshot();
  });
});
