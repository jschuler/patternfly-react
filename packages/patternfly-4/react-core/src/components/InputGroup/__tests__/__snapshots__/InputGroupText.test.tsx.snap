// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputGroupText test InputGroupText should match snapshot 1`] = `
<[object Object]
  className="pf-c-input-group__text ''"
>
  <div>
    ReactNode
  </div>
</[object Object]>
`;
